### PR TITLE
React 16

### PR DESCRIPTION
**Pull request by sodiumjoe**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- ~~[ ] One core team UX approval~~

### CODE REVIEW

File: tests/constants.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 // Constants for tests
 
-export const MOSTLY_STABLE_DELAY = 64;
+export const MOSTLY_STABLE_DELAY = 1064;
```
Comment (jondlm) [inline tests/constants.js]: Now _more even mostly stable_ goodness!

Comment (sodiumjoe) [reply tests/constants.js]: oh, should i try turning that back down

File: src/components/SplitVertical/SplitVertical.spec.jsx [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 import assert from 'assert';
-import sinon from 'sinon';
 import _ from 'lodash';
 import { common } from '../../util/generic-tests';
 import SplitVertical from './SplitVertical';
 import DragCaptureZone from '../DragCaptureZone/DragCaptureZone';
 import { Motion } from 'react-motion';
 import { MOSTLY_STABLE_DELAY } from '../../../tests/constants';
 
+jest.setTimeout(10000);
```
Comment (ogupte) [inline src/components/SplitVertical/SplitVertical.spec.jsx]: why?

File: tests/constants.js (hunk 1 above)
Comment (ogupte) [reply tests/constants.js]: why is it so high?

Comment (sodiumjoe) [reply tests/constants.js]: not sure, i'll try turning it back down

File: src/components/SplitVertical/SplitVertical.spec.jsx (hunk 2 above)
Comment (sodiumjoe) [reply src/components/SplitVertical/SplitVertical.spec.jsx]: i'll try removing that too, i was running into the timeout

**Comment by jondlm** [comment]
Might be worth taking the extra time to test cross browser on this one.

**Pull request merged**